flow/connectors/bigquery/merge_statement_generator.go

```diff
@@ -47,6 +47,9 @@ func (m *mergeStmtGenerator) generateFlattenedCTE() string {
 			// if the type is JSON, then just extract JSON
 			castStmt = fmt.Sprintf("CAST(PARSE_JSON(JSON_VALUE(_peerdb_data, '$.%s'),wide_number_mode=>'round') AS %s) AS `%s`",
 				colName, bqType, colName)
+		case qvalue.QValueKindHStore:
+			castStmt = fmt.Sprintf("CAST(PARSE_JSON(CONCAT('{', REPLACE(JSON_VALUE(_peerdb_data, '$.%s'), '=>', ':'), '}')) as %s) AS `%s`",
```

Comment:
This will break if people use `=>` in their hstore. Also `NULL` needs to be changed to `null`